scripts/install-autoshutdown-server-extension/on-jupyter-server-start.sh

```diff
@@ -49,12 +49,12 @@ wget -O .auto-shutdown/extension.tar.gz https://github.com/aws-samples/sagemaker
 # aws s3 --endpoint-url [S3 Interface Endpoint] cp s3://[tarball location] .
 
 # Installs the extension
-cd .auto-shutdown
-tar xzf extension.tar.gz
-cd sagemaker_studio_autoshutdown-0.1.1
-pip install pydeps/*
```

Comment:
As discussed offline, we don't need to disable and enable extensions here.  
All we need to make this work is remove this line:
```
pip install pydeps/*
```

This installs dependencies which conflict with Studio JupyterServer dependencies. Required dependencies for this extension are already available from Studio, we don't need this install here.

Reply:
I would call out that this seems to make things a bit more fragile in that the extension may no longer introduce any Python dependencies that aren't already present in the base image, but happy to go along with it if that's the preferred fix.

Tested & it works for me - PR updated now